packages/lab/stories/form-field-next/form-field-next.stories.tsx

```diff
@@ -118,21 +106,26 @@ export const WithControls: ComponentStory<typeof FormFieldNext> = (props) => {
 export const MultiInput: ComponentStory<typeof FormFieldNext> = (props) => {
```

Comment:
Could we merge somehow the WithControls and MultiInput? Seems to me they're quite similar.